OutOfSchool/OutOfSchool.WebApi/Models/Parent/ParentBlockByAdminDto.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+using OutOfSchool.WebApi.Validators;
+
+namespace OutOfSchool.WebApi.Models.Parent;
+
+public class ParentBlockByAdminDto
+{
+    [Required]
+    public Guid ParentId { get; set; }
+
+    [Required]
+    public bool IsBlocked { get; set; }
```

Comment:
(Naming) I don't get how `IsBlocked` could be `false` for the entity named `ParentBlockByAdminDto` (and controller action named `BlockParent`)?

Probably it should be `...ToggleBlock...` or something similar in name of both to show that it also could **unblock**?

Reply:
I came up with the names BlockStatus and BlockAction, but ToggleBlock fits better, so I used it. Thank you!

Reply:
Sorry for being not specific, but I was more expecting for changes in clas/method names, not property name.
Reasoning: 
- `ToggleBlock = false` is not always clear if it means do not toggle or toggle to unblocked status
- at same time `BlockParent()` - usually is treated as block only.

Please reconsider your changes

Reply:
@VadymLevkovskyi I propose BlockUnblockParent or UpdateParentBlockStatus.

Reply:
As you wish, just do it for both controller method name (currently `BlockParent()`) and class name (currently `ParentBlockByAdminDto`), while keeping `IsBlocked` property as is.

Reply:
Fixed. Thank you!